CONTRIBUTING.md

```diff
@@ -0,0 +1,35 @@
+# Contribution to Coronavirus Tracker API
+
+First off, thanks for taking the time to contribute!
+Every commit supports the open source ecosystem in case of [COVID-19](https://en.wikipedia.org/wiki/2019%E2%80%9320_coronavirus_pandemic).
+
+## Testing
+
+We have a handful of unit tests to cover most of functions.
+Please write new test cases for new code you create.
+
+## Submitting changes
+
+* If you're unable to find an open issue, [open a new one](https://github.com/ExpDev07/coronavirus-tracker-api/issues/new). Be sure to include a **title and clear description**, as much relevant information as possible
+* Open  a new [GitHub Pull Request to coronavirus-tracker-api](https://github.com/ExpDev07/coronavirus-tracker-api/pulls) with a clear list of what you've done (read more about [pull requests](http://help.github.com/pull-requests/)). Include the relevant issue number if applicable.
+* We will love you forever if you include unit tests. We can always use more test coverage
+
+## Your First Code Contribution
+
+Unsure where to begin contributing to coronavirus-tracker-api ? You can start by looking through these issues labels:
+
+* [Enhancement issues](https://github.com/ExpDev07/coronavirus-tracker-api/labels/enhancement) - issues for new feature or request
+* [Help wanted issues](https://github.com/ExpDev07/coronavirus-tracker-api/labels/help%20wanted) -  extra attention is needed
+* [Documentation issues](https://github.com/ExpDev07/coronavirus-tracker-api/labels/documentation) - improvements or additions to documentation
+
+## Styleguide
+
+Please follow [PEP8](https://www.python.org/dev/peps/pep-0008/) guide.
+See [Running Test](./README.md#running-tests) and [Linting](./README.md#linting) for further instructions.
+
```

Comment:
Should we add a step here about running the formatter?
`pipenv run fmt` or `make fmt` or `black . -l 120`

Reply:
Thanks for feedback. Added `make ftm` to styleguide section.